src/index.html

```diff
@@ -11,8 +12,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="iMac"
+      />
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="card__score">
+        <div class="card__score card__score--stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="card__score card__score--review">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price card__price--price-title">Price:</div>
+        <div class="card__price card__price--price-value">$2,199</div>
```

Comment:
```suggestion
        <div class="card__price---title">Price:</div>
        <div class="card__price---value">$2,199</div>
```

Reply:
This suggestion doesn't work as tests says that wrong BEM. 
<img width="892" alt="Screenshot 2024-05-17 at 12 40 47" src="https://github.com/mate-academy/layout_product-cards/assets/108335373/fbaee926-0e0b-4204-aa76-6afa6ef79090">